File: Lista_Geral_Python/comandos de entrada e saida/question10.py
### Escreva um programa que leia o valor dos dois catetos de um
# triângulo retângulo e  calcule o valor da hipotenusa.
from math import hypot


cateto_oposto = int(input("Insira o valor do cateto oposto: "))
cateto_adjaente = int(input("Insira o valor do cateto adjacente: "))
print(f"O cateto oposto tem valor de: {cateto_oposto}cm\nO cateto adjacente tem valor de: {cateto_adjaente}cm\nA hipotenusa tem valor de: {(hypot(cateto_adjaente, cateto_oposto)):.2f}cm")

File: Lista_Geral_C/comandos de entrada e saida/question10.c
// Escreva um programa que leia o valor dos dois catetos de um triângulo retângulo e  
// calcule o valor da hipotenusa.
#include <stdio.h>
#include <math.h>

int main(){

    int cateto_oposto, cateto_adjacente;
    float hipotenusa;

    printf("Informe o valor do cateto oposto: ");
    scanf("%d",&cateto_oposto);
    printf("Informe o valor do cateto adjacente: ");
    scanf("%d",&cateto_adjacente);

    hipotenusa = sqrt(cateto_adjacente * cateto_adjacente + cateto_oposto * cateto_oposto);

    printf("O valor da hipotenusa e: %.1f", hipotenusa);

    return 0;
}